docs/upgrades/manual.md

```diff
@@ -69,5 +69,4 @@ To upgrade K3s manually, you can download the desired version of the K3s binary
 
 1. Download the desired version of the K3s binary from [releases](https://github.com/k3s-io/k3s/releases)
 2. Copy the downloaded binary to `/usr/local/bin/k3s` (or your desired location)
-3. Stop the old k3s binary
-4. Launch the new k3s binary
\ No newline at end of file
+3. Restart the k3s or k3s-agent service
```

Comment:
```suggestion
3. Restart the k3s or k3s-agent service (systemd/openrc)
    OR
    Restart the k3s process (binary)
```

RKE2 does not operate as a standalone binary, but K3s very much can. So we need to to include both options. We just talked about the binary in the previous steps.

Reply:
They both function the same when run from a terminal. I really hope that noone is running k3s from a shell. Honestly I would like to revise the rest of the docs to no longer suggest just running `k3s server` to start a node.

Reply:
I mean.... isn't that what our k3s docker image does, run K3s in a shell, not a service?

Reply:
Yes, but also you can't upgrade the k3s image by replacing the binary and killing the process.